### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by mimipeshy**
#### Milestone: Weather App
When creating the project the following points have been added

- [ ] setup Webpack for this repo
- [ ] implemented API calls using promises.
- [ ]  added a catch to the promise so errors will be handled
- [ ]  styled the pages using CSS

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "weather-app",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "build": "webpack",
+    "watch": "webpack --watch",
+    "start": "webpack-dev-server --open"
```
Comment (christianotieno) [inline package.json]: ## Improve functionality
- [x] great job adding the start script @mimipeshy, but it is not working as t has not been set properly, Would you mind fixing this? Thank you!

File: src/view.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  data.unit === 'metric' ? 'glass__select--active' : ''
+}" data-unit="metric">C</div>
+          <div class="glass__select fahrenheit ${
+  data.unit === 'imperial' ? 'glass__select--active' : ''
+}" data-unit="imperial">F</div>
+      </div>
+    </div>
+    `;
+
+    // this._bigCircle.style.backgroundImage = `url(${data.flag})`;
```
Comment (christianotieno) [inline src/view.js]: ## Remove commented out code
- [x] It’s advisable these deleted. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

**Review by christianotieno — CHANGES_REQUESTED** [review]
## Changes Required ♻️ 
Hello @mimipeshy 👋 
The weather app looks pretty great, the main functionality is working as expected and good job with the great design, keep it up. 👏 
We have some few issues to take care of before we proceed, 👇 

### Required
- [x] We have some informing errors in the console belong to the TypeError groups of errors. Let's fix this.

### Happy coding 💻 

#### Christian Otieno (CR)

**Review by peterrobert — APPROVED** [review]
### APPROVED 🥳 👍🏿 💯 

Hello @mimipeshy, very impressive work. I love your design. Your work is perfect 💯 . Good work making the changes as requested. 

### on to the next project and HAPPY CODING 💻

**Pull request merged**